pkg/client/simple/vfsclientset/instancegroup.go

```diff
@@ -27,7 +27,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/klog"
-	"k8s.io/kops/pkg/apis/kops"
```

Comment:
kopsapi "k8s.io/kops/pkg/apis/kops" i see the alias in the other packages , so recommend kopsapi  i think , but it is ok.

Reply:
I agree, `api` is too vague so if we switched the `k8s.io/kops/pkg/apis/kops` import in this file and `cloud_instance_group.go` to use `kopsapi` that could be more informative.

Reply:
I am going to update.